tests/unit/components/cylc/tree.vue.spec.js

```diff
@@ -0,0 +1,88 @@
+// we mount the tree to include the TreeItem component and other vuetify children components
+import { mount } from '@vue/test-utils'
+import { expect } from 'chai'
+// import vuetify here so that we do not have warnings in the console output
+// eslint-disable-next-line no-unused-vars
+import * as vuetify from '@/plugins/vuetify'
+import Tree from '@/components/cylc/Tree'
+import { simpleWorkflowTree4Nodes } from './tree.data'
+
+const cycles = new Map([
+  ['user/workflow1', new Set([
+    '20100101T0000Z'
+  ]
+  )]
```

Comment:
The closing parenthesis is on the wrong line for logical grouping here, which made me think there was a syntax error initially.

Reply:
Fixed! Though simply moving the `)]` back one indentation level breaks the linter. Had to move the `)` up, and separate from the `]`. Now it looks like:

```js
const cycles = new Map([
  [
    'user/workflow1',
    new Set([
      '20100101T0000Z'
    ])
  ]
])
```